README.md

```diff
@@ -40,7 +44,7 @@ The `-R` parameter specifies the directory of the translation rule while `-r` sp
 Run the below python script to format the result so each log entry is on its own line.
 
 ```
-python3 singleLine.py data/o365/o365_translated.json data/  o365_ocsf.json
+python3 singleLine.py data/o365/o365_translated.json o365_ocsf.json
```

Comment:
Seems `data/` was in the way? I was getting an error message until i removed it:

```command
# python3 singleLine.py data/o365/o365_translated.json data/  o365_ocsf.json
Usage: python singleLine.py input_file output_file
```